README.md

```diff
@@ -13,7 +13,15 @@ The Diagnostic Log and Trace Viewer is an application that can send and receive
 ### Command line usage;
 
 + Windows: `dlt_viewer.exe [OPTIONS]`
-+ Linux: ```dlt_viewer [OPTIONS] Options: -h Print usage (only in Linux) -p projectfile Loading project file on startup (must end with .dlp) -l logfile Loading logfile on startup (must end with .dlt) -f filterfile Loading filterfile on startup (must end with .dlf) -c logfile textfile Convert logfile file to textfile (logfile must end with .dlt) -e "name of plugin|command in plugin|param1|param2|paramn..." Execute command in a command interface plugin. -s Enable silent mode without warning message boxes```
++ Linux: `dlt_viewer [OPTIONS]`
+
+Options: 
++ `-h` Print usage (only in Linux)
++ `-p projectfile` Loading project file on startup (must end with .dlp)
++ `-l logfile` Loading logfile on startup (must end with .dlt)
++ `-f filterfile` Loading filterfile on startup (must end with .dlf)
++ `-c logfile textfile` Convert logfile file to textfile (logfile must end with .dlt)
++ `-e "name of plugin|command in plugin|param1|param2|paramn..."` Execute command in a command interface plugin. -s Enable silent mode without warning message boxes```
```

Comment:
I guess `-s` should also be on its own line

Reply:
@tobsan 
Sure, I missed it. Thanks!